iohyve

```diff
@@ -282,27 +298,144 @@ __list() {
 		else
 			boot="NO"
 		fi
-		local description="$(zfs get -H -o value iohyve:description $pool/iohyve/$g)"
-                printf "%s^%s^%s^%s^%s\n" "$g" "$vmm" "$running" "$boot" "$description"
+		# Print common section (guest, vmm, running, rcboot)
+		printf "%s^%s^%s^%s" "$g" "$vmm" "$running" "$boot" 
+		
+		# Get variable and print for verbose section (tap, con)
+		if [ -n "$(echo "$flags" | grep 'v')" ]; then
+			local tap="$(zfs get -H -o value iohyve:tap $pool/iohyve/$g)"
+			local con="$(zfs get -H -o value iohyve:con $pool/iohyve/$g)"
+			
+			printf "^%s^%s" "$tap" "$con"
+		fi
+		
+		# Get description property and print
+		if [ -n "$(echo "$flags" | grep 'd')" ]; then
+			local description="$(zfs get -H -o value iohyve:description $pool/iohyve/$g)"
+			printf "^%s" "$description"
+		fi
+		
+		printf "\n" #Print new line for next guest.
             done
 	done
-	) | column -ts^
+	) | column -ts^ | \
```

Comment:
This was inspired by the function above piping to `column`. I figured the same could be done for a smaller bit of code needing to get piped.